python/GaelZllHbb13TeVconfig/general.ini

```diff
@@ -272,6 +280,6 @@ weightF_bTagJES_UP:     <!Weights|weight_noBTag!> * bTagWeight_JESUp
 weightF_bTagJES_DOWN:     <!Weights|weight_noBTag!> * bTagWeight_JESDown
 
 ;(PU)
-weightF_pileUp_UP: <!Weights|GenSign!> * puWeightUp * <!Weights|BTagWeight!> * <!Weights|weightF_EWK!> * <!Weights|weightF_muSF!>
-weightF_pileUp_DOWN: <!Weights|GenSign!> * puWeightDown * <!Weights|BTagWeight!> * <!Weights|weightF_EWK!> * <!Weights|weightF_muSF!>
+weightF_pileUp_UP: <!Weights|weight_noPU!>* puWeightUp
```

Comment:
@GLP90 shuoldn't this variation contain all the other (central) weights like btag and EWK corrections?